packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml

```
@@ -103,6 +342,14 @@ processors:
       type: long
       ignore_missing: true
       field: source.port
+  - convert:
+      field: panw.panos.source.port
+      type: long
+      ignore_failure: true
```

Assuming the mapping type is a `long`, then leaving the bad field value present by `ignore_failure: true` will result in an event that cannot be indexed (the only exception is if the value is a `null`). IMO it's better to deal with those issues in the pipeline than let an even pass through that will fail to index. My suggestion is to remove the field with an on_failure, or if you don't want a silent failure you can remove the field and append an error.message.